### PR TITLE
Cb 4 navbar

### PR DESCRIPTION
**Pull request by sulemanbutta**
* **Please check if the PR fulfills these requirements**
- [X] The pull request title contains the ticket number in the title. For example [CB-1]
Yes

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
Adds navbar to components section (new feature)

* **What is the current behavior?What does the code do before you made the new changes?** 
This is a new component added.

* **What is the new behavior?**
Contains links to different pages and displays them to the user

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
Should not require any changes.

* **Other information**:

### CODE REVIEW

File: frontend/src/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                    <NavLink href="https://github.com/Codubee/ExploreCitiesTTH4-6">GitHub</NavLink>
+                </NavItem>
+                <NavItem>
+                     <NavLink href="/userinput/">User input</NavLink>
+                </NavItem>
+                <NavItem>
+                    <NavLink href="/landingpage/">Landing Page</NavLink>
+                </NavItem>
+            </Nav>
+        <NavbarText>Codubee</NavbarText>
```
Comment (JerryTeng01) [inline frontend/src/components/Navbar/Navbar.js]: We could also add a link to the Codubee website here.

**Review by JerryTeng01 — COMMENTED** [review]
Hey Suleman, great work on the navbar! I really like the aesthetics of it so far. I think our ideas of how to structure the website might be a little different, so we can discuss that and where each of the components will be at our next meeting. Nice job!

Edit: I just reread the description for your task and it cleared up some stuff for me. We can still discuss some more structuring guidelines at our meeting

**Comment by Aarushi-Pandey** [comment]
Hey Suleman, great job on the Navbar! I really like the aesthetics and color scheme so far. Like Jerry said, adding a link to Codubee would be a great idea. I think we should discuss the names of the NavItem components in our next meeting and possibly adding the links to the Explanation page, Team page, Tools Used page, and Features page. Since this wasn't mentioned in the description of the component in Jira, I understand why you didn't include them yet. Other than that I love it. Keep up the good work!

**Comment by sulemanbutta** [comment]
> Hey Suleman, great job on the Navbar! I really like the aesthetics and color scheme so far. Like Jerry said, adding a link to Codubee would be a great idea. I think we should discuss the names of the NavItem components in our next meeting and possibly adding the links to the Explanation page, Team page, Tools Used page, and Features page. Since this wasn't mentioned in the description of the component in Jira, I understand why you didn't include them yet. Other than that I love it. Keep up the good work!

I was under the impression that the Explanation, Team, Tools , and Features would all be apart of the Landing Page. However we probably could change this if we wanted to. Thanks for the feedback.

**Comment by JerryTeng01** [comment]
I think the components would be in the landing page too. It will be more apparent once the routing is completed.

**Comment by sulemanbutta** [comment]
I added the link to the Codubee and removed some the comments. Thanks for the feedback.

**Review by JerryTeng01 — APPROVED** [review]
Looks good to me! 👍

**Review by gustavooviedo — APPROVED** [review]
Hey Suleman, your component looks great! I really like the layout of your component. Looks ready to merge!

**Pull request merged**